pulp_rpm/app/tasks/publishing.py

```diff
@@ -695,6 +696,11 @@ def generate_repo_metadata(
         )
         sign_results = signing_service.sign(repomd_path)
 
+        signature_file_path = sign_results["signature"]
+        if Path(signature_file_path).stat().st_size == 0:
```

Comment:
Use `os.stat(signature_file_path).st_size` to avoid bringing in `pathlib`

I don't mind pathlib in some cases but in this case it's not any simpler.  Also pathlib has some performance issues.

Reply:
@dralley updated :)